dune/xt/grid/gridprovider/cube.hh

```diff
@@ -139,7 +139,8 @@ public:
          const FieldVector<typename GridType::ctype, GridType::dimension>& upper_right,
          const std::array<unsigned int, GridType::dimension>& num_elements,
          const unsigned int num_refinements,
-         const std::array<unsigned int, GridType::dimension>& overlap_size)
+         const std::array<unsigned int, GridType::dimension>& overlap_size,
+         MPIHelper::MPICommunicator mpi_comm)
```

Comment:
Could you also add the default `= MPIHelper::getCommunicator()` here (as below in the `create` methods) and in the other places, or are there reasons against?

Reply:
Well, it would be inconsistent. No create method has defaults for anything, only the make_* do.

Reply:
True! I just thought about backwards compatibility. But we do not really call these create methods manually anyway, right?